perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -15,55 +17,83 @@ set -e
 sudo apt-get update
 echo Installing git
 sudo apt-get install git
-echo Installing python3-pip
-sudo apt-get -y install python3-pip
-echo Installing libraries to run python script
-pip install google-cloud
-pip install google-cloud-vision
-pip install google-api-python-client
-pip install prettytable
 echo Installing go-lang  1.20.5
 wget -O go_tar.tar.gz https://go.dev/dl/go1.20.5.linux-amd64.tar.gz
```

Comment:
Can we use `-q` to silent the progress of `wget` command?

Ref: https://stackoverflow.com/questions/4686464/how-can-i-show-the-wget-progress-bar-only